Cargo.toml

```diff
@@ -59,6 +63,8 @@ ssd1306 = "0.9.0"
 display-interface = "0.5.0"
 embedded-graphics = "0.8.1"
 mpu6050 = "0.1.6"
+eyeball = "0.7.0"
+smol = "2.0.2"
```

Comment:
The way versioning works in rust, these versions require at least the version specified, and will increment the least significant portion of the build. So Eyeball could pull 0.7.1 and be happy.